### DISCUSSION TITLE
Keycloak Swagger-UI Authorization Code flow

### DISCUSSION THREAD

**Discussion by Extollite** [quarkusio/quarkus#27554] [Q&A]
Hi,
I am using oidc, keycloak-authorization and smallrye-openapi dependencies.
My keycloak properties are looking like that:
```
quarkus.swagger-ui.always-include=true
quarkus.smallrye-openapi.info-title=Example API

%prod.quarkus.oidc.auth-server-url=https://<REDACTED>/auth/realms/<REDACTED>-office
quarkus.oidc.auth-server-url=https://<REDACTED>/auth/realms/<REDACTED>-office
quarkus.oidc.client-id=<CLIENT_ID>
%prod.quarkus.oidc.credentials.secret=8aee7f00-2f9a-47d8-bd55-5e581bbbfa18
quarkus.oidc.credentials.secret=020d4101-c759-432a-a2ac-150b8e15a7ce
```

Can I manually set only available Flow to `Authorization Code` and to have `client_id` filled up automatically?
<img width="695" alt="Screenshot 2022-08-28 at 13 50 58" src="https://user-images.githubusercontent.com/42713788/187072919-6e5f041c-3142-46f1-8471-224487e72564.png">

[a comment by an automated account is not shown]

**Comment by sberyozkin** [comment]
The code flow is not integrated with Swagger UI AFAIK

**Reply by Extollite** [reply]
I was able to make it show all possible Flows by using @SecuritySchemes annotation, but I don't know how to make only Authorization Code to show up.

**Reply by phillip-kruger** [reply]
Yes Swagger UI always show all. If you keen you can log a feature request with Swagger UI
